src/v/kafka/server/tests/delete_topics_test.cc

```diff
@@ -108,8 +108,8 @@ class delete_topics_request_fixture : public redpanda_thread_fixture {
     void validate_topic_is_deleteted(const model::topic& tp) {
         kafka::metadata_response resp = get_topic_metadata(tp);
         auto it = std::find_if(
-          std::cbegin(resp.data.topics),
-          std::cend(resp.data.topics),
+          resp.data.topics.begin(),
```

Comment:
This is because `fragmented_vector` is missing `const`-qualified versions of begin & end?

Reply:
I believe you cannot compare const iterator to non const in frag vector, and you can in vector, so I just switch this all to use the non constant version (in the assertion below)